components/eamxx/tests/coupled/dynamics_physics/homme_shoc_cld_spa_p3_rrtmgp_pg2/homme_shoc_cld_spa_p3_rrtmgp_pg2_gll_output.yaml

```diff
@@ -0,0 +1,26 @@
+%YAML 1.1
```

Comment:
We should be able to write both gll and pg2 fields in the same file (restart already does that). Perhaps we should do it here too, to have one more instance of 2+ grids fields written on the same mesh?

Reply:
Yes, I'll combine. I did this because ncvis doesn't know how to output nicely when `ncol_d` is used (instead of `ncol`) and I was checking output. Didn't think to switch it back.

Reply:
No, I'll combine. Most people doing actual output are using better tools.